Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -51,12 +51,13 @@ def runSelected(opt):
                      8, #BH/Cosmic MC
                      25, #MC ttbar
                      4.22, #cosmic data
+                     4.53, #run1 data + miniAOD
                      1000, #data+prompt
                      1001, #data+express
                      136.731, #2016B Photon data
+                     136.7611, #2016E JetHT reMINIAOD from 80X legacy
```

Comment:
as this PR is now, this does not work, not sure whether we really need for the purpose of this PR

Reply:
@fabiocos removed